renesas/FreeRTOS-Kernel
# Introduce portMEMORY_BARRIER(), prvTaskExitError() and xPortIsInsideInterrupt() into RX generic port

Description
-----------
This PR fixes issue #8 and #7 and also introduces xPortIsInsideInterrupt() into RX generic port similaly such as Cortex-Mxx ports.

Test Steps
-----------
(1) It was tested by full demo of RX72N Envision Kit with CC-RX/ICCRX/GUNURX.
(2) Additionally full demos of TB-RX130/RX231/RX65N with CC-RX/ICCRX/GUNURX were executed to check that there were no problems.

Somehow GitHub complains that "Can't automatically merge.".

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
